payment/payment.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+  <section class="payment">
+    <div>
+      <h2>Billing details</h2>
+      <form class="payment-form">
+        <label for="Name"><input type="text" placeholder="Name" name="Name" class="payment-form__input"></label>
+        <label for="Last name"><input type="text" placeholder="Last Name" name="Last name" class="payment-form__input"></label>
+        <label for="ID number"><input type="text" placeholder="ID Number" name="ID Number" class="payment-form__input"></label>
```

Comment:
Mae, el input necesita un atributo ID, que es el que va a vincularse con el for del Label, y acordate que va sin espacios.. "last-name" por ejemplo